app/ports/templates/ports/port-detail/installation_stats.html

```diff
@@ -1,5 +1,24 @@
 {% load sort_version %}
 
+<br>
+<form class="bg-light p-2 form form-inline justify-content-center">
+    <lable for="days">Displaying statistics for users who made submissions in previous:</lable>
+    <select onchange="installationStatsAjax()" id="days" name="days" class="format-control mx-2">
+        <option value="7" {% if days == 7 %}selected{% endif %}>7 Days</option>
+        <option value="30" {% if days == 30 %}selected{% endif %}>30 Days</option>
+        <option value="90" {% if days == 90 %}selected{% endif %}>90 Days</option>
+        <option value="180" {% if days == 180 %}selected{% endif %}>180 Days</option>
+        <option value="365" {% if days == 365 %}selected{% endif %}>365 Days</option>
```

Comment:
This kind of repetition seems a bit error prone when you will want to add the next option.

It could be nice if one could change the duration in URL. Of course this will get a bit tricky to select the correct option if user provides something that's not on the menu (you would need to dynamically add one more option as the currently selected one).

Reply:
Shall we restrict the user to use only some values (7, 30, 90, 180, 365)? We could then get rid of the repetition also.